### ISSUE TITLE
Allow podboat file naming to use article feed title instead of query title

### ISSUE THREAD

**Issue by Aicaya** [newsboat/newsboat#428]
**Newsboat version**: 2.14.0

Steps to reproduce the issue:

1. Create query to group podcasts in `url`:
```
https://somepodcast.com/feed "!Feed A" podcasts
https://anotherpodcast.com/rss "!Feed B" podcasts
"query:Podcasts:tags # \"podcasts\""
```

2. Change `download-filename-format` to use `%n`. Method used in my config:
```
download-filename-format "%n/%F - %t.%e"
```

3. Enqueue from within query feed, download via podboat.

With the settings above, if I remove the ! to unhide the podcast from the main index and enqueue straight from the feed itself, podboat creates a folder under the podcast's feed title and places the 
episode there. When enqueuing from the query feed, however, it creates a "Podcasts" folder and drops everything in one place regardless of source:
```
/podcasts
|-- Feed A
|   |-- 2019-01-01 - episode 1.mp3
|   `-- 2019-01-08 - episode 2.mp3
|-- Feed B
|   `-- 2019-01-15 - some title.ogg
`-- Podcasts
    |-- 2019-01-15 - episode 3.mp3
    |-- 2019-01-22 - another title.ogg
    `-- 2019-01-22 - episode 4.mp3
```

So basically, I'd like to be able to keep all podcasts under a single query feed to separate them from all the text stuff more neatly while still being able to label and organize the downloaded files under their respective titles. Didn't seen an option on the documentation's format strings that differentiated between query title and feed title, so here I am.

**Comment by Minoru** [comment]
> Didn't seen an option on the documentation's format strings that differentiated between query title and feed title

There are none indeed! I agree this is an omission that ought to be fixed. Thanks for a well-written issue, @Aicaya.

We already have `t`/`T` [in `articlelist-format`](https://newsboat.org/releases/2.14/docs/newsboat.html#idm3826), and I should we should add `N` to [`download-path` and `download-filename-format`](https://newsboat.org/releases/2.14/docs/newsboat.html#idm4018).

This is a pretty easy change to make, so I'm tagging this as good first issue. PRs are welcome!
